crates/storage/codecs/derive/src/lib.rs

```diff
@@ -1,9 +1,15 @@
-#![cfg_attr(docsrs, feature(doc_cfg))]
+//! Derive macros for the Compact codec traits.
+
 #![doc(
     html_logo_url = "https://raw.githubusercontent.com/paradigmxyz/reth/main/assets/reth-docs.png",
     html_favicon_url = "https://avatars0.githubusercontent.com/u/97369466?s=256",
     issue_tracker_base_url = "https://github.com/paradigmxzy/reth/issues/"
 )]
+// TODO: add these warnings back
+// #![warn(missing_debug_implementations, missing_docs, unreachable_pub, rustdoc::all)]
```

Comment:
is this supposed to be added back in this pr?

Reply:
"back" is wrong here, I meant they should be added in the future since they were not present before this PR